go/vt/vttablet/tabletserver/schema/engine.go

```diff
@@ -706,7 +706,11 @@ func (se *Engine) RegisterNotifier(name string, f notifier, runNotifier bool) {
 		created = append(created, table)
 	}
 	if runNotifier {
-		f(se.tables, created, nil, nil)
+		s := make(map[string]*Table, len(se.tables))
+		for k, v := range se.tables {
+			s[k] = v
+		}
```

Comment:
```suggestion
		s := maps.Clone(se.tables)
```

Reply:
We were doing this in 2 more places. I fixed those too.